cordova-common/src/ConfigChanges/ConfigFile.js

```diff
@@ -193,6 +193,11 @@ function resolveConfigFilePath(project_dir, platform, file) {
         return filepath;
     }
 
+    // FIXME android studio
+    if(platform === 'android' && !fs.existsSync(filepath)) {
```

Comment:
Can you put a comment here to describe how this works and why it is needed.
It looks to me like if we are looking for config.xml and platform is android, the `return filepath;` call on ln:193 is going to return before ever getting here.